internal/db/nosql/cassandra/cassandra.go

```diff
@@ -266,18 +267,24 @@ func (c *client) Query(stmt string, names []string) *Queryx {
 	return gocqlx.Query(c.session.Query(stmt), names)
 }
 
+func clientError(err error, keys ...string) error {
```

Comment:
![badge](https://img.shields.io/badge/review-imo-orange.svg?style=flat-square)
the function name does not represent its role.
in this case, it should be "wrapWithKeys", "errorWrappedWithKeys", or "errWithKeys".